src/plugins/interaction/SimpleCollision/SimpleCollision.cpp

```diff
@@ -44,6 +46,7 @@
 
 #include <limits>
 #include <memory>
+#include <iostream>
 
```

Comment:
are all of these includes necessary? It looks like the file otherwise didn't change.

Reply:
left over during debug, removed.